src/content/getting-started/designers/designers.mdx

```diff
@@ -2,120 +2,179 @@
+</ImageComponent>
 
-![Update indicator screenshot](images/designers-5.png)
+All designs should start with the 2x Grid template. Once you have successfully [saved the grid](#gat-the-kit) as a template, navigate to `File → New file from Template` and select the grid template.
 
-</ImageComponent>
+#### Structure
+
+At the core of the 2x Grid concept is the idea of divisions of two. This can be applied to most surfaces, 3-D objects, and architecture as a means to divide space into a helpful grid system. 
 
-### Download the kit from GitHub
+Dividing your space into 2, 4, 8, 16, 32, or 64 columns gives you the basis of the system with the purpose of helping you make decisions and organize your content.
 
-This is the best method for one-time downloads of the kit, or if you are looking for previous versions.
+You can see a full breakdown of the 2x Grid at [IBM.com/design](https://www.ibm.com/design/language/elements/2x-grid).
 
-#### `1.` Download the Carbon Design Kit
+### Symbols
 
-- Go to the <a href="https://github.com/carbon-design-system/carbon-design-kit" target="_blank">Carbon Design Kit</a> repo and click the folder for latest version of the Carbon Design Kit.
-- On the next page click `Carbon Design Kit.sketch`
-- Click the “View Raw” link to begin downloading the kit.
+*Symbols* are Sketch versions of Carbon [components](/components/overview), [add-ons](/contributing/add-ons), and [icons](/guidelines/iconography/library).
 
-<ImageComponent cols="12" caption="Carbon Design Kit repo">
+<ImageComponent cols="8" caption="IDL symbol menu navigation">
 
-![Carbon Design Kit repo screenshot](images/designers-2_new.png)
+![Carbon symbol menu navigation](/images/IDL-kit-library.png)
 
 </ImageComponent>
 
-#### `2.` Open the Carbon Design Kit in Sketch
 
-- Open and use the kit like any other Sketch file.
-- Use the left panel in Sketch to navigate through the different pages of the kit.
 
-<ImageComponent cols="12" caption="Design kit file">
+There are two kinds of symbols — `library symbols` and `document symbols`. Library symbols are available in any Sketch document, while document symbols are specific to the document in which they are found.
+
+From the `Insert` menu, select `Symbols → Carbon Design System` to add symbols onto your page.
+
+Carbon symbols are built to be flexible, and designers should not detach symbols from the library. Once a symbol is detached, you will no longer receive updates as they are released.
+
+#### Structure
 
-![Kit file screenshot](images/designers-3.png)
+Symbols are organized by component; after selecting a component, you'll see the variations, pieces and states of that component. 
+
+- To add the whole component, simply locate the variant you want and drag it into the work space.
+
+- For even more configurablility, use the individual building blocks under "Pieces." Be sure your custom design adheres to our [design guidelines](/guidelines/accessibility/overview).
```

Comment:
The building blocks are under "items" not "pieces." Pieces are like tiny micro parts of a component symbol. These are the best [definitions](https://github.ibm.com/duo/design-kit/wiki/Terminology) but they might help clear it up for you a bit.

Reply:
Changed this too!